blocks/library/embed/index.js

```diff
@@ -22,6 +26,23 @@ registerBlock( 'core/embed', {
 	edit( { attributes, setAttributes, focus, setFocus } ) {
 		const { url, title, caption } = attributes;
 
+		if ( ! url ) {
```

Comment:
Oh yeah, I misunderstood. We should do that at some point for sure.  I don't think we need to right now, but once the placeholder actually has to do something we will probably turn it into its own component.